v2.0/demo-json-support.md

```diff
@@ -0,0 +1,229 @@
+~~~
+
+Set the database as the default:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SET DATABASE = jsonb_test;
+~~~
+
+Then [grant privileges](grant.html) to the `maxroach` user:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> GRANT ALL ON DATABASE jsonb_test TO maxroach;
+~~~
+
+## Step 5. Create a table
+
+Still in the SQL shell, create a table called `programming`:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> CREATE TABLE programming (
+    id UUID DEFAULT uuid_v4()::UUID PRIMARY KEY,
+    posts JSONB
+  );
+~~~
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SHOW CREATE TABLE programming;
+~~~
+~~~
++--------------+-------------------------------------------------+
+|    Table     |                   CreateTable                   |
++--------------+-------------------------------------------------+
+| programming  | CREATE TABLE programming (                      |
+|              |     id UUID NOT NULL DEFAULT uuid_v4()::UUID,   |
+|              |     posts JSON NULL,                            |
+|              |     CONSTRAINT "primary" PRIMARY KEY (id ASC),  |
+|              |     FAMILY "primary" (id, posts)                |
+|              | )                                               |
++--------------+-------------------------------------------------+
+~~~
+
+## Step 6. Run the code
+
+Now that you have a database, user, and a table, let's run code to insert rows into the table.
+
+The code queries the [Reddit API](https://www.reddit.com/dev/api/) for posts in [/r/programming](https://www.reddit.com/r/programming/). The Reddit API only returns 25 results per page; however, each page returns an `"after"` string that tells you how to get the next page. Therefore, the program does the following in a loop:
+
+1. Makes a request to the API.
+2. Inserts the results into the table and grabs the `"after"` string.
+3. Uses the new `"after"` string as the basis for the next request.
+
+Download the <a href="https://raw.githubusercontent.com/cockroachdb/docs/master/_includes/json/json-sample.go" download><code>json-sample.go</code></a> file, or create the file yourself and copy the code into it:
+
+{% include copy-clipboard.html %}
+~~~ go
+{% include json/json-sample.go %}
+~~~
+
+In a new terminal window, navigate to your sample code file and run it:
+
+{% include copy-clipboard.html %}
+~~~ shell
+$ go run json_sample.go
+~~~
+
+The program will take awhile to finish, but you can start querying the data right away with the [built-in SQL client](use-the-built-in-sql-client.html):
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT count(*) FROM programming;
+~~~
+~~~
++-------+
+| count |
++-------+
+|  4100 |
++-------+
+~~~
+
+## Step 7. Query the data
+
+Back in the terminal where the SQL shell is running, verify that rows of data are being inserted into your table:
+
+{% include copy-clipboard.html %}
+~~~ sql
+> SELECT count(*) FROM programming;
+~~~
+~~~
++-------+
+| count |
++-------+
+|  7500 |
```

Comment:
If someone does this soon after starting the app, the count is going to be a lot lower. Maybe use 200 and 250 here?